lib/inputs/java/static.ts

```diff
@@ -2,13 +2,15 @@ import * as path from "path";
 import { ExtractAction } from "../../extractor/types";
 import { streamToSha1 } from "../../stream-utils";
 
-const ignoredPaths = [path.normalize("/usr/lib")];
+const ignoredPaths = ["/usr/lib", "gradle/caches"];
```

Comment:
why not just `gradle/cache`? It will also catch `gradle/caches`

Reply:
No good reason :) Thanks for flagging, changed it